command-line/src/cmd-test/test.api

```diff
@@ -11,4 +13,8 @@ service {
     response 'Entity'
   }
 
+  delete '/profile/{id}' spec {
+    name 'Delete profile'
```

Comment:
@PaulTaykalo it's a very simple example of supported method spec: response body is missing in this case.
You just get 204 with nothing.